src/chester.js

```diff
@@ -119,9 +120,10 @@ module.exports.Chester = (secret: string | Buffer, {
   };
 
   return {
-    _crypter: crypter,
+    _crypter: crypter, // why is this exposed?
```

Comment:
Mostly to make it easier to test and to inspect. E.g. if someone wanted to analyse the data without considering expiration.

Reply:
In that case they could create a separate instance of Crypter with the same config, no?

Reply:
That's true. However the creation of the secret for the Crypter requires a Buffer. So it's not straight forward, but certainly possible. Will adjust in test

Reply:
removed